### PR TITLE
LOG-13509 remove support info

### PR DESCRIPTION
**Pull request by rsheng-r7**

### CODE REVIEW

File: LE.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
 EOD
 
   gem.authors	= ["Mark Lacomber", "Stephen Hynes"]
```
Comment (rjacobs-r7) [inline LE.gemspec]:
```suggestion
  gem.authors	= ["Rapid7"]
```

**Comment by sbialkowski-r7** [comment]
> We should fix the CI

Completely deprecated library (`this library is deprecated, and will no longer be updated`)
https://github.com/rapid7/le_ruby#deprecated

don't see a reason why we should fix it

**Comment by rsheng-r7** [comment]
@rjacobs-r7 @sbialkowski-r7 shall i ignore the CI and merge? or shall i just close the PR? or fix the CI and merge and release(how to fix the CI)?

**Comment by sbialkowski-r7** [comment]
I stand by my comment, shouldn't fix CI since we advertise that the library is deprecated and will not be updated, also waste of time. Waiting for @rjacobs-r7 to weigh in

**Comment by rjacobs-r7** [comment]
The 10 second fix for the CI in this case would just be to change the bundler version in travis.yml (if you click the Travis link it tells you exactly which one you need). If you both feel like it's not worth it then feel free to go ahead.

**Pull request closed without merging**